### PR TITLE
chore: fix linting of test-app and remove webkit from e2e testing

### PR DESCRIPTION
**Pull request by ThisIsMissEm**

### CODE REVIEW

File: e2e/browser/test-app/next-env.d.ts [hunk 1]
```diff
@@ -1,23 +1,5 @@
...
-//
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
-// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
-// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
-// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
-// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-//
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
```
Comment (ThisIsMissEm) [inline e2e/browser/test-app/next-env.d.ts:2]: @chelseapinka @NSeydoux this file was broken by the recent changes, specifically, we need to exclude these files from license-header and linting/prettier.

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "lint": "npm run lint:check",
     "lint:fix": "npm run lint:eslint -- --fix && npm run lint:prettier -- --write",
     "lint:check": "npm run lint:eslint && npm run lint:prettier -- --check",
-    "lint:eslint": "eslint --config .eslintrc.js \"src/\" \"e2e/\" --ignore-pattern \"e2e/**/*js\"",
```
Comment (ThisIsMissEm) [inline package.json:16]: This is done with a new rule in `.eslintignore` and `.prettierignore`

File: playwright.config.ts [hunk 3]
```diff
@@ -60,15 +60,17 @@ const config: PlaywrightTestConfig = {
...
+    //
+    // {
+    //   name: "WebKit",
+    //   use: {
+    //     browserName: "webkit",
+    //     userAgent: `Browser-based solid-client end-to-end tests running ${
+    //       process.env.CI === "true" ? "in CI" : "locally"
+    //     }. Mozilla/5.0 (iPhone; CPU iPhone OS 13_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.1 Mobile/15E148 Safari/604.1`,
+    //   },
+    // },
```
Comment (ThisIsMissEm) [inline playwright.config.ts:73]: Keeping it here just in case we want to get it back after ESS is fixed.

**Review by solid-akb — APPROVED** [review]

File: .prettierignore [hunk 4]
```diff
@@ -5,3 +5,4 @@ docs/api/source/api
 *.snap
 *.env
 *.env.*
+**/.next/
```
Comment (chelseapinka) [inline .prettierignore:8]: missing newline her

Comment (ThisIsMissEm) [reply .prettierignore:8]: eh, ooops, we can fix it later.

**Pull request merged**